### ISSUE TITLE
Dense point cloud and links on the website

### ISSUE THREAD

**Issue by JudyYe** [epic-kitchens/epic-fields-code#7]
Hi team,

Great work! Congratulations! 

Could you please explain the relation between the colmap commands in Issue#3 and the raw COLMAP format from EPIC-FIELDS website? More specifically, the two links on the website the output or the input to the 3 colmap commands? Thanks! 

> The dense registered frames in raw COLMAP format can be found at: [here (133G)](https://www.dropbox.com/scl/fo/8li0s3fku8aeaeymd7vqr/h?rlkey=oalzl6cw877d71ljun2ai6iwm&dl=0).
The sparse frames including raw COLMAP database can be found at: [here (91.6G)](https://www.dropbox.com/scl/fo/0wtphqqyp4fu6bd7dhbfs/h?rlkey=ju21graeixi6vpecrf7rqurpt&dl=0).

_Originally posted by @zhifanzhu in https://github.com/epic-kitchens/epic-fields-code/issues/3#issuecomment-1792348561_

**Comment by zhifanzhu** [comment]
The sparse (91.6G) means sparse cameras + sparse point clouds.
The dense registered (133G) means dense cameras + sparse point clouds.

The input to those 3 colmap commands shall be the sparse (91.6G), and the 3 colmap commands will output dense point clouds.
In other words, you need to run those 3 commands yourself to get the dense point clouds. :)

**Comment by JudyYe** [comment]
Hi Zhifan,

Thank you for your quick response to my rookie question! 

Let me rephrase to confirm. To get dense pointclouds in teaser, I need to feed sparse (91.6G) to 3 colmap commands. The dense (133G) is not used along the way. 

I assume the `images/` need to be downloaded from EPIC-KITCHENS website individually? 

Thanks!

**Comment by dimadamen** [comment]
Yes that is correct. These annotations are for the already published EPIC-KITCHENS dataset which you need to download separately as downscaled images or full videos.
